var/spack/repos/builtin/packages/exago/package.py

```diff
@@ -45,6 +47,8 @@ class Exago(CMakePackage, CudaPackage, ROCmPackage):
     conflicts(
         "+python", when="+ipopt+rocm", msg="Python bindings require -fPIC with Ipopt for rocm."
     )
+    variant("tests", default=True, description="Enable/Disable Testing")
```

Comment:
Can we remove this variant and use `self.run_tests` ?

Reply:
I am unfamiliar with the usage pattern. Instead of `self.define_from_variant` to configure the CMake variable `EXAGO_RUN_TESTS`, would I just define that based on `self.run_tests` instead?

I am not fully familiar with usage patterns here, but this would be nice as a spack-based alternative. Key here is we want to avoid building tests altogether, as some testing builds can be buggy...

We do have both unit and functionality tests, so perhaps a more complex configuration as in umpire would be appropriate if we want to support that, but `self.run_tests` seems like it would work just fine here.